app/import_functions.go

```diff
@@ -1541,12 +1541,7 @@ func (a *App) uploadAttachments(c *request.Context, attachments *[]AttachmentImp
 		attachment := attachment
 		fileInfo, err := a.importAttachment(c, &attachment, post, teamID)
 		if err != nil {
-			if attachment.Path != nil {
-				mlog.Warn(
-					"failed to import attachment",
-					mlog.String("path", *attachment.Path),
-					mlog.String("error", err.Error()))
-			} else {
+			if attachment.Path == nil {
```

Comment:
initially we were checking that attachment Path cannot be nil now its opposite why is that

Reply:
because we don't want to warn again about those missing attachments as we have already warned about those in `processAttachmentPaths' function in `import.go`

Reply:
because we don't want to warn again about those missing attachments as we have already warned about those in `processAttachmentPaths' function in `import.go`